src/lnp/application/payment/types.rs

```diff
@@ -256,3 +261,134 @@ impl DumbDefault for TempChannelId {
         Self(Default::default())
     }
 }
+
+#[derive(Wrapper, Clone, Debug, From, PartialEq, Eq)]
+pub struct NodeColor([u8; 3]);
+
+impl LightningEncode for NodeColor {
+    fn lightning_encode<E: io::Write>(
+        &self,
+        mut e: E,
+    ) -> Result<usize, io::Error> {
+        let len = e.write(self.as_inner())?;
+        Ok(len)
+    }
+}
+
+impl LightningDecode for NodeColor {
+    fn lightning_decode<D: io::Read>(mut d: D) -> Result<Self, LightningError> {
+        let mut buf = [0u8; 3];
+        d.read_exact(&mut buf)?;
+        Ok(Self::from_inner(buf))
+    }
+}
+
+#[derive(Wrapper, Clone, Debug, From, PartialEq, Eq)]
+pub struct Alias([u8; 3]);
+
+impl LightningEncode for Alias {
```

Comment:
The same like here for `NodeColor`

Reply:
Defining Alias with`Slice32` as same as `TempChannelId`.